packages/google_maps_flutter/google_maps_flutter_web/lib/src/third_party/to_screen_location/to_screen_location.dart

```diff
@@ -0,0 +1,57 @@
+// The MIT License (MIT)
+//
```

Comment:
I don't think these first two lines should be here; I've never seen an actual MIT license file that was labelled here like this.

See also https://opensource.org/licenses/MIT

Reply:
D'oh, copied verbatim from here: https://krasimirtsonev.com/license, didn't notice that "The MIT License (MIT)" was decoration and not part of the license itself. I'll fix.

Reply:
There's division about this. Left the title in, respecting the original author's license text.